web/packages/build/package.json

```diff
@@ -62,8 +62,7 @@
     "babel-loader": "^8.2.5",
     "clean-webpack-plugin": "4.0.0",
     "core-js": "^3",
-    "cross-env": "5.0.5",
-    "cross-spawn": "6.0.5",
+    "cross-spawn": "7.0.3",
```

Comment:
`cross-spawn` is not needed as well. ;) It's used in `g-build.js` and `g-start.js` in `web/packages/build/bin`. Those are old scripts used back when Webpack was used to build Teleport and Connect. AFAIK the scripts can be completely removed, along with their uses in package.json and readmes.

```
$ rg g-start -w
web/packages/build/package.json
13:    "g-start": "./bin/g-start.js",

$ rg g-build -w
web/packages/build/package.json
14:    "g-build": "./bin/g-build.js"

web/packages/build/README.md
37:$ yarn g-build
59:$ yarn g-build --config webpack.config.js
```